### ISSUE TITLE
wrong field dimensions

### ISSUE THREAD

**Issue by Raminzr** [HuguesTHOMAS/KPConv-PyTorch#215]
Hello Thomas,
first of all thanks for your amazing work, I'm using your tools for processing and detecting the objects in a 3D point clouds from geometry photography, but I tried first to run the code on S3DIS and SemanticKitti datasets and maybe later on SensatUrban, to be sure about its functionality and learn about your methods. after running then training on both datasets S3DIS and SemanticKitti I would run your test_model.py scripts to get results for the test set. for both it has worked but there is a difference between them, by testing on SemanticKitti, the results will be saved as ply data successfully with the help of write_ply.py you provided. on the other hand it doesn't work on S3DIS for saving the results as ply and it reported an error as wrong field dimensions. I tried to print out the shape and dimension of both input (Points, Preds) for write_ply function and may it's the reason for this error, I will be thankful to you for your answers and help 
best regards Ramin
<img width="871" alt="Bildschirm­foto 2023-03-16 um 15 05 32" src="https://user-images.githubusercontent.com/89695141/233942522-2bfddc7e-5ab5-4969-b006-d65e8e1ff17e.png">
<img width="824" alt="Bildschirmfoto 2023-04-13 um 12 12 06" src="https://user-images.githubusercontent.com/89695141/233941788-6f8788ed-6282-4219-b45b-9741120e151f.png">

**Comment by MohamedMansour89** [comment]
Hi, 

i have the same issue, did you solve it ? 
thanks

**Comment by MohamedMansour89** [comment]
actually  i solved it. The issue in the paths of the vaildation set Area_5 (78719063 points) and the potential points (622587 points). The paths in Linux and Windows are defined differently, i.e slash /  and backslash \.    That means by evaluating the validation dataset during training the potential points replaces the original validation set Area_5, due to the line   , 
cloud_name = file_path.split('/')[-1]  in trainer.py , 
. Simply change every slash to ('\\') and it will work , assuming you are on windows operating system.

**Comment by Raminzr** [comment]
Hi, I didn't solve this Issue because I hat to stop the project using KPConv, but I've read about that, it was same as you find as solution.
best regard
